merkle/rust/native_jolt/guest/src/lib.rs

```diff
@@ -1,47 +1,10 @@
 // #![cfg_attr(feature = "guest")]
 #![no_main]
 
-use sha2::{Sha256, Digest};
+use wasm::merkelize;
```

Comment:
reuse the implementation in `wasm/lib` instead of re-implementing here.

Reply:
Is it working with Jolt, when importing?